src/main/java/org/takes/rq/RqForm.java

```diff
@@ -55,17 +55,25 @@
  * decorator before passing request to this class.
  *
  * <p>The class is immutable and thread-safe.
- *
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
- * @since 0.9
  * @link <a href="http://www.w3.org/TR/html401/interact/forms.html">Forms in HTML</a>
  * @see org.takes.rq.RqGreedy
+ * @since 0.9
  */
 @SuppressWarnings("PMD.TooManyMethods")
 public interface RqForm extends Request {
 
     /**
+     * The parameter separator.
+     */
+    String PARAM_SEPARATOR = "&";
```

Comment:
@yegor256 sorry, feeling stupid now... but If I declare them inside classes as private Qulice complains about having those strings twice in the same file, if I keep them in the interface they are clearly accessible from the outside... I'm not able to think about anything else... a part from excluding some Qulice check